README.md

```diff
@@ -1,116 +1,194 @@
+#### Scenario
 
-<a name="resources-mock"/>
 
-### Mock Interview Resources
+Problem Statement TODO :bug:
 
-If you're looking for forums to practice interviewing and communication skills, 
-these resources are a good place to start. 
+Suppose an array sorted in ascending order is rotated at some pivot unknown to you beforehand.
 
-* Fellow students, friends, alumni, and LinkedIn connections. Reach out and ask 
-    if these people would be willing to give you some time to do a mock interview. 
-    The worst answer you can get is "no".
-* [PraMP](https://www.pramp.com/): free for 6 sessions[\*\*](#footnotes)
-* [interviewing.io](https://interviewing.io): free[\*\*](#footnotes)
-* [UWB-ACM](https://uwbacm.com/): annual Mock Interview event, free for all 
-    attendees. Check the club's event calendar for upcoming dates.
+(i.e., [0,1,2,4,5,6,7] might become [4,5,6,7,0,1,2]).
 
-<a name="resume"/>
+You are given a target value to search. If found in the array return its index, otherwise return -1.
 
-## Resume Resources
+You may assume no duplicate exists in the array.
 
-Revising and polishing your resume is a little bit easier with these resources 
-and guides.
+Your algorithm's runtime complexity must be in the order of O(log n).
 
-* [CareerCup Resume Guide](https://careercup.com/resume): written by the 
-    author of CTCI, this resume guide breaks down the structure, content, 
-    and best practices of Software Engineer resumes.
-* [X, Y, Z Formula and Other Google Application Advice](https://www.inc.com/bill-murphy-jr/google-recruiters-say-these-5-resume-tips-including-x-y-z-formula-will-improve-your-odds-of-getting-hired-at-google.html):
-    best practices and company-specific advice that helps your resume 
-    make an impression on recruiters and engineers (at Google and elsewhere).
+#### Example Input 1
+Input: nums = [4,5,6,7,0,1,2], target = 0
+Output: 4
 
-Make sure you get other humans to review your resume drafts for issues and 
-points of interest; a second pair of eyes is your best defense against typos.
+#### Example Input 2
 
-<a name="footnotes"/>
+Input: nums = [4,5,6,7,0,1,2], target = 3
+Output: -1
 
-## Footnotes
+If the problem is simple enough, remove this section. TODO :bug:
 
-\* The contributors haven't used this platform before; YMMV.
+#### Function Signature
+class Solution {
+public:
+    int search(vector<int>& nums, int target) {
+        int left = 0, right = nums.size() - 1;
+        while (left <= right) {
+            int mid = left + (right - left) / 2;
+            if (nums[mid] == target) return mid;
+            if (nums[mid] >= nums[left]) {
+                if (nums[left] <= target && nums[mid] > target) right = mid - 1;
+                else left = mid + 1;
+            } else {
+                if (nums[mid] < target && nums[right] >= target) left = mid + 1;
+                else right = mid - 1;
+            }
+        }
+        return -1;
+    }
+};
```

Comment:
There are two halves of the README; the first half is for the problem statement (not including the solution code), and the second half is for the solution. This method would be written up in the Solution section, not the Problem section.